src/fields/OembedField.php

```diff
@@ -153,6 +154,8 @@ public function getSettingsHtml()
      */
     public function getInputHtml($value, ElementInterface $element = null): string
     {
+        Craft::$app->view->registerAssetBundle(OembedAsset::class);
```

Comment:
Can I confirm this doesn't load multiple scripts?

Reply:
Sure. You can either verify this by having multiple oembed fields on the same page or by just duplicating the line so `registerAssetBundle` is called multiple times.